nginx_conf.sh

```diff
@@ -1,28 +1,34 @@
 #!/bin/bash
 
-# Updating and installing required commands
-apt update  -y
+# installing required commands
 apt install wget -y
 apt install curl -y
 
 # initiating variables
-URL=https://transfer.sh/QX7UuX/bootcamp_aca.conf
-response=$(curl --write-out %{http_code} --silent --output /dev/null $URL)
-
+url='https://transfer.sh/QX7UuX/bootcamp_aca.conf'
+response=$(curl --write-out %{http_code} --silent --output /dev/null $url)
+file="bootcamp_aca.conf"
 # Checking if the given URL works and if we already have one
 if [[ $response = 200 ]]; then
-	wget $URL
+	wget $url
+fi
+
+# chechking if file is zero or not 
+if [[ -z $file ]]; then 
+       mv bootcamp_aca.conf /etc/nginx/sites-enabled/
 fi
 
-# checking if we already have the copy as our nginx default conf
-deff=$(diff bootcamp_aca.conf /etc/nginx/sites-enabled/default)
+#remove /etc/nginx/sites-enabled/default file
+
+if [[ -f /etc/nginx/sites-enable/default ]]
+then
+rm /etc/nginx/sites-enabled/default
 
-if [[ $deff -ne 0 ]]; then 
-       cp bootcamp_aca.conf /etc/nginx/sites-enabled/default
 fi
 
+#testing nginx
+nginx -t
```

Comment:
you should check if the error code is 1 or 0 (`echo $?`)
if 0 -> then everything is OK
if 1 -> nginx testing failed

Reply:
done

Reply:
do `else exit`